typing/path.mli

```diff
@@ -57,11 +57,19 @@ and extra_ty =
   *)
 
 val same: t -> t -> bool
+val equiv: t -> t -> bool
```

Comment:
Could you have a documentation comment that explains the difference between these two functions, for example by pointing to `Ident.equiv`?